docroot/themes/humsci/humsci_basic/src/js/shared/carousel-slides/carousel-slides-height.js

```diff
@@ -1,18 +1,20 @@
-const slides = document.querySelectorAll('.paragraph--type--hs-carousel');
+// This work below applies uniform height to both the Hero Layered Slider (formerly Carousel) and 
+// the Hero Gradient Slider paragraph component slides.
+const slides = document.querySelectorAll('.paragraph--type--hs-carousel, .ptype-hs-gradient-hero-slider');
```

Comment:
The class of `ptype-hs-gradient-hero-slider` only targets the sliders within the main content area. Can we update this to `paragraph--type--hs-gradient-hero-slider` which wraps the sliders in the hero and main content areas?

Reply:
Regarding the centering of the text mentioned above, this wasn't addressed in the implementation or acceptance criteria within the JIRA card that scoped the work in this PR, so we should let H&S aware and then address in future work should they desire the content to be centered.

Reply:
This all makes sense to me, the class has been updated as suggested.

I have an idea of what should be added if H&S were to want the content centered, so we can handle that once we've discussed with them.